README.md

```diff
@@ -21,6 +21,7 @@ This repository contains the following projects:
  - `java-client`: the Java language binding
  - `scala-client`: the Scala language binding
  - `kotlin-client`: the Kotlin language binding
+ - `tracing-observation`: module to integrate with [Micrometer Observation](https://micrometer.io/docs/observation)
```

Comment:
Actually we don't know what that will do because it all depends on the configuration of the handler mechanism. I added the tracing prefix because i'm using the tracing api.